sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -317,7 +317,7 @@ export interface IssuerProperties {
 }
 
 // @public
-export type KeyUsageType = 'digitalSignature' | 'nonRepudiation' | 'keyEncipherment' | 'dataEncipherment' | 'keyAgreement' | 'keyCertSign' | 'cRLSign' | 'encipherOnly' | 'decipherOnly';
+export type KeyUsageType = string;
```

Comment:
Please let's make this a set of string literals. One can end with `| string` if necessary, though I would rather avoid having public API changes as part of this PR if possible.

Reply:
Same as above